Source/Core/Core/PowerPC/PPCAnalyst.h

```diff
@@ -230,6 +230,6 @@ class PPCAnalyzer
 
 void LogFunctionCall(u32 addr);
 void FindFunctions(u32 startAddr, u32 endAddr, PPCSymbolDB* func_db);
-bool AnalyzeFunction(u32 startAddr, Symbol& func, int max_size = 0);
+bool AnalyzeFunction(u32 startAddr, Symbol& func, int max_size = 0, bool reanalyze = false);
```

Comment:
This is something that should go in its own function, ReanalyzeFunction, like so:

```cpp
bool ReanalyzeFunction(u32 start_addr, Symbol& func, int max_size)
{
  assert(func.analyzed);

  func.analyzed = false;
  return AnalyzeFunction(start_addr, func, max_size);
}
```

also I'd like to re-emphasize avoiding using bools as parameters where reasonably possible. It's annoying both for review and understanding what goes on at a call-site (it's also not strongly-typed).

i.e.

```cpp
SomeFunc(name, other_name, true);
```

communicates nothing other than yet another ambiguity.